src/services/TrackingService.ts

```diff
@@ -175,7 +175,7 @@ const getTotalPlantsInZones = async (organizationId: number, siteId: number): Pr
       'plantingSubzones.fullName',
       'plantingSubzones.populations.species_scientificName',
       'plantingSubzones.populations.species_organization_id',
-      'plantingSubzones.populations.totalPlants',
+      'plantingSubzones.populations.totalPlants(raw)',
```

Comment:
there might be some v1 code depending on this

Reply:
oh, i'll check into it. Thanks

Reply:
I found an instance of this and fixed it. Thanks for the callout

Reply:
Awesome, thanks @charlie-tf 👍